src/acl/InnerNode.cc

```diff
@@ -21,7 +21,8 @@
 void
 Acl::InnerNode::prepareForUse()
 {
-    std::for_each(nodes.begin(), nodes.end(), std::mem_fun(&ACL::prepareForUse));
+    for (auto &node : nodes)
```

Comment:
Please remove the reference. AFAICT, it is not needed now, it might slow down code now, and it might even result in subtle bugs if the `nodes` type changes to use reference counting (an existing TODO).